tests/tests.rs

```diff
@@ -1,36 +1,40 @@
 use std::time::Duration;
 
-use async_std::{prelude::*, task, sync::channel};
+use async_std::prelude::*;
```

Comment:
@flub I was wondering why the change in `use` syntax here?  Is it not better to use `{}` syntax to make all these imports on 1 line?

Reply:
mixing multiple levels of of imports quickly becomes confusing to me, that is i avoid having to use `::` inside `{ }` and nesting braces for imports.  fewer lines isn't always more readable.  but this doesn't matter much does it?

Reply:
No, just a style question and I was simply curious.

Reply:
the downside to this however is that it makes the diff harder to read

Reply:
if you stick to that style all following diffs are more readable.  If you don't every single non-trivial diff becomes harder to read.  this is all bikeshedding of course and depends on personal taste :)  i wish cargo-fmt had much stricter formatting rules around imports instead of allowing so many variations, it would avoid this bikeshedding.